### ISSUE TITLE
Fix docstring in UniversalPicklingError

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8627]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)| @markrwilliams reported|
|-|-|
|Trac ID|trac#8627|
|Type|defect|
|Created|2016-07-19 06:15:05Z|

The `_UniveralPicklingError` in the patch that closed [#8504](https://github.com/twisted/twisted/issues/8504) has an invalid docstring.  It refers to `PicklingException` in `cPickle` and `pickle`, but no such exception exists.  It should be `PicklingError` instead.

<details><summary>Searchable metadata</summary>

```
trac-id__8627 8627
type__defect defect
reporter__markrwilliams markrwilliams
priority__normal normal
milestone__PyPy_support PyPy-support
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468908905233381 1468908905233381
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 958ecfe2894daf9f58f530f8adf8de0b2a69f33c
```
#!CommitTicketReference repository="" revision="958ecfe2894daf9f58f530f8adf8de0b2a69f33c"
Merge pull request #361 from twisted/8627-markrwilliams-universalpicklingerror-docstring

Author: markrwilliams
Reviewer: rodrigc
Fixes: #8627

Fix docstring in Pickle tests
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
